src/main/resources/lang/en-GB.json

```diff
@@ -926,6 +926,368 @@
     "permissionmode":{
       "most_restrictive": "most restrictive",
       "hierarchical": "hierarchical"
+    },
+    "moldogevent": {
+      "emote_created": {
+        "select": "emote_created",
+        "display": "Emote created"
```

Comment:
Not sure about this `select`, `display` stuff 🤔

Reply:
Do you have a better way to handle this? I need both as I need the select for the list, and for actual selecting. I need the display for sending the event.